### PR TITLE
External db map fix

### PR DESCRIPTION
**Pull request by vsitnik**

### CODE REVIEW

File: config/external_db_map/default.txt [hunk 1]
```diff
@@ -133,11 +139,16 @@ GenBank	GenBank_translation	translation
...
 GenBank_translation	GenBank_translation	translation
 
+HAMAP	HAMAP	translation
+
 INTERPRO	Interpro	translation
 
 MEROPS	MEROPS	translation
 
 PDB	PDB	translation
+PFAM	PFAM	translation
```
Comment (ens-LCampbell) [inline config/external_db_map/default.txt:149]: Just wondering why such mapping only added now ? Would of expected PFAM in place

File: config/external_db_map/default.txt [hunk 2]
```diff
@@ -88,6 +91,9 @@ GenBank	GenBank_transcript	transcript
 GB	GenBank_transcript	transcript
 GenBank_transcript	GenBank_transcript	transcript
 
+JGIDB	JGI_TRANSCRIPT	transcript
```
Comment (ens-LCampbell) [inline config/external_db_map/default.txt:94]: Thanks !! This DB was the only failure for the 8 new genome loads Im currently running

**Review by ens-LCampbell — APPROVED** [review]
Seems ok to me, thanks again

File: config/external_db_map/default.txt (hunk 1 above)
Comment (vsitnik) [reply config/external_db_map/default.txt:149]: PFAM is in place, but we just allowing it to be attached only to the peptide/CDS/translation xrefs

**Pull request merged**